src/pages/run-a-node.tsx

```diff
@@ -862,7 +862,7 @@ const RunANodePage = () => {
         </Column>
       </StakingCalloutContainer>
       <Content>
-        <H3 id="plan-on-staking">
+        <H3 id="plan-on-staking" sx={{ display: 'flex', alignItems: 'center' }}>
```

Comment:
We can use [style props](https://chakra-ui.com/docs/styled-system/style-props) here to simplify 😄 

```suggestion
        <H3 id="plan-on-staking" display="flex" alignItems="center">
```